<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>795</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1117</width>
    <height>795</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1117</width>
    <height>795</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ваш ресторан</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#f9f3e1;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>501</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>font: 75 italic 15pt &quot;Ubuntu Condensed&quot;;
border: 1px solid gray</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="layout_cheks">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lable_chek">
         <property name="styleSheet">
          <string notr="true">qproperty-alignment:'AlignVCenter | AlignCenter';
border: 0px solid gray;
font: 75 italic 14pt &quot;Yrsa&quot;;
</string>
         </property>
         <property name="text">
          <string>Ваш Заказ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lable_view_check">
         <property name="text">
          <string>Нименование                        Кол-во                            Цена</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textCheck">
         <property name="styleSheet">
          <string notr="true">border: 0px solid gray;</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu Condensed'; font-size:15pt; font-weight:72; font-style:italic;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_order_summ">
           <property name="text">
            <string>Итого:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Order">
       <property name="text">
        <string>Сделать Заказ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>661</width>
      <height>721</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>657</width>
          <height>717</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QLabel" name="label_onedish">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>661</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 italic 16pt &quot;Ubuntu Condensed&quot;;
border: 1px solid gray;
qproperty-alignment:'AlignVCenter | AlignCenter';</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>641</width>
           <height>151</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="Layout_onedish">
          <property name="spacing">
           <number>5</number>
          </property>
         </layout>
        </widget>
        <widget class="QLabel" name="label_twodish">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>180</y>
           <width>661</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string>font: 75 italic 16pt &quot;Ubuntu Condensed&quot;;
border: 1px solid gray;
qproperty-alignment:'AlignVCenter | AlignCenter';</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>641</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="Layout_twodish">
          <property name="spacing">
           <number>5</number>
          </property>
         </layout>
        </widget>
        <widget class="QLabel" name="label_threedish">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>390</y>
           <width>661</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 italic 16pt &quot;Ubuntu Condensed&quot;;
border: 1px solid gray;
qproperty-alignment:'AlignVCenter | AlignCenter';</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>420</y>
           <width>641</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="Layout_threedish">
          <property name="spacing">
           <number>5</number>
          </property>
         </layout>
        </widget>
        <widget class="QLabel" name="label_drinks">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>590</y>
           <width>661</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 italic 16pt &quot;Ubuntu Condensed&quot;;
border: 1px solid gray;
qproperty-alignment:'AlignVCenter | AlignCenter';</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>620</y>
           <width>641</width>
           <height>101</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="Layout_drinks">
          <property name="spacing">
           <number>5</number>
          </property>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_menu">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>0</y>
      <width>1091</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-alignment:'AlignVCenter | AlignCenter';
font: 75 italic 16pt &quot;Yrsa&quot;;
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Выход</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_3">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Режим Администратора</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
